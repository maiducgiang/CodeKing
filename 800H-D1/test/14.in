10
75
28 72 62 66 40 69 41 46 47 50 38 46 34 75 54 17 17 18 20 65 52 12 64 60 36 71 56 27 26 45 56 38 13 26 16 25 40 62 27 65 46 16 18 61 1 49 3 18 69 17 67 43 35 4 72 30 8 16 74 22 27 16 40 66 42 43 60 9 62 58 23 23 33 58 71 
76
9 55 73 37 12 15 59 73 65 29 13 19 21 43 58 13 52 12 64 40 27 1 8 17 65 29 31 3 9 50 62 27 63 20 71 47 67 34 70 76 1 27 7 72 73 20 9 29 36 7 61 41 12 20 61 19 76 72 73 21 44 66 37 70 59 50 24 40 35 22 52 55 73 68 20 28 
98
90 81 69 1 36 45 80 12 66 87 98 86 70 19 38 58 96 43 80 32 9 40 21 36 50 21 87 90 36 32 96 6 44 74 34 34 27 72 2 53 72 83 84 75 37 55 48 75 31 63 27 47 80 95 15 29 35 1 89 65 16 70 27 90 36 58 53 48 15 12 20 37 69 85 15 36 56 55 78 11 62 7 38 3 56 98 7 20 37 55 9 95 9 51 90 19 28 6 
87
66 65 33 72 38 47 39 67 50 17 33 71 74 51 18 87 73 86 4 18 74 40 85 71 43 24 58 74 9 2 87 4 30 82 1 8 85 78 14 62 83 29 42 85 74 43 68 64 60 40 9 54 25 57 49 55 83 86 72 36 50 31 41 16 32 43 75 76 57 52 26 80 32 49 78 74 31 74 45 47 78 48 32 64 38 83 75 
79
2 14 63 42 22 31 23 54 7 52 38 60 73 16 75 10 65 67 40 42 13 74 9 7 22 20 71 78 10 70 45 12 1 13 42 78 18 18 3 71 78 54 49 74 44 31 10 68 20 16 19 77 50 17 26 45 28 36 19 25 32 67 72 46 24 63 76 72 72 63 29 65 79 60 66 15 45 20 71 
96
84 26 83 64 43 53 27 7 49 26 96 21 15 78 71 74 47 23 7 42 40 72 24 37 42 58 80 37 20 23 86 87 24 43 64 34 17 86 94 2 75 1 77 56 12 76 89 5 6 59 28 59 64 37 95 86 3 72 67 54 24 54 11 48 15 14 52 23 82 77 63 8 6 16 7 11 47 86 46 96 75 41 88 42 39 88 33 40 85 65 15 39 7 85 61 49 
97
14 9 62 15 89 66 77 73 25 79 67 41 73 47 64 88 82 71 96 22 88 15 78 75 14 93 17 65 29 16 81 39 26 5 34 78 8 35 58 26 95 43 7 22 69 6 55 80 92 47 39 94 39 38 5 84 48 26 43 35 35 12 32 49 77 44 15 64 46 19 75 10 12 5 96 91 52 67 60 15 69 57 73 35 14 95 68 9 18 54 67 28 7 69 7 92 48 
77
55 70 41 4 11 68 55 60 76 5 9 73 43 75 31 74 14 9 41 2 4 62 23 33 4 40 13 56 41 71 17 14 14 61 48 11 30 60 49 33 3 75 28 16 16 10 33 51 70 68 43 10 48 76 69 63 10 22 17 36 18 44 9 10 20 10 53 10 56 70 71 63 36 47 59 12 42 
82
7 59 13 60 20 71 7 50 56 12 7 57 74 21 28 41 19 40 13 76 25 32 55 48 67 48 19 14 41 58 54 1 31 60 37 42 52 49 18 7 15 80 22 22 20 16 71 73 71 61 82 11 17 1 16 16 44 1 81 72 22 21 42 68 73 29 21 49 3 70 79 39 46 23 32 57 24 5 76 15 77 81 
94
88 68 80 14 5 53 46 12 14 52 14 63 78 42 74 94 17 51 90 39 31 23 19 81 19 13 9 70 46 54 60 35 47 30 3 36 52 4 90 53 79 40 19 40 68 35 76 48 31 34 29 38 86 80 1 10 55 79 74 64 16 78 92 20 28 75 93 82 62 58 53 84 74 31 1 20 33 88 3 53 50 84 53 26 89 13 12 50 25 51 33 75 67 48 
