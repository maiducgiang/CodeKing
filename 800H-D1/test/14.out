45
36
41
35
40
55
56
36
32
36
